src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write your code here
+
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+
+const population = document.getElementsByClassName('population');
+
+const data = [];
+
+for (const el of population) {
+  const tmp = el.innerText;
```

Comment:
It's generally a good practice to use `textContent` instead of `innerText` unless you specifically need the layout-aware aspects of `innerText`. `textContent` is faster as it doesn't trigger a reflow.